controllers/contactsController.js

```diff
@@ -1,120 +1,89 @@
-const {
-  getContactsService,
-  getContactByIdService,
-  addContactService,
-  removeContactService,
-  updateContactService,
-  updateStatusContactService,
-} = require("../services/contactsService");
+const { Contacts } = require("../models/Contacts");
+const { HttpError } = require("../utils/HttpError");
 const { controllerWrapper } = require("../utils/controllerWrapper");
 
-const getContacts = controllerWrapper(async (_, res) => {
-  const contacts = await getContactsService();
-  return res.json(contacts);
+const getContacts = controllerWrapper(async (req, res) => {
+  const { _id: owner } = req.user;
+
+  const { page = 1, limit = 20, favorite } = req.query;
+  const skip = (page - 1) * limit;
+  const filter = { owner };
+
+  if (favorite && favorite === "true") {
+    filter.favorite = true;
+  }
+
+  if (favorite && favorite === "false") {
```

Comment:
```suggestion
  if (favorite === "false") {
```

if it's undefined it won't be equal to 'false' anyway

Reply:
if (!favorite) {
    filter.favorite = false;
  }
  
  and like this one?

Reply:
No, it should be like this if (favorite === "false") {... 
Because query parameter is not required